src/Storage/WpCache.php

```diff
@@ -32,6 +32,7 @@ public function set( $key, $data, $expire ) {
 			$key,
 			is_array( $data ) ? $data : $data->toArray(),
 			$this->group_name,
+			// phpcs:ignore WordPressVIPMinimum.Performance.LowExpiryCacheTime.CacheTimeUndetermined
```

Comment:
I'm sure there are ways to satisfy phpcs on this $expire paramter. Or just ignore the warning and move on.